packages/@react-spectrum/s2/src/ComboBox.tsx

```diff
@@ -384,11 +500,52 @@ export function ComboBoxSection<T extends object>(props: ComboBoxSectionProps<T>
   return (
     <>
       <AriaListBoxSection
-        {...props}
-        className={section({size})}>
+        {...props}>
         {props.children}
       </AriaListBoxSection>
-      <Divider />
+      <Divider size={size} />
     </>
   );
 }
+
+export function Divider(props: SeparatorProps & {size?: 'S' | 'M' | 'L' | 'XL' | undefined}): ReactNode {
+  return (
+    <Separator
+      {...props}
+      className={mergeStyles(
+        divider({
```

Comment:
`divider` only supports `'S' | 'M' | 'L'`, if XL is sent in, it'll treat it like M

Reply:
Hm...actually looking back on it, we explicitly set the size to always be 'M' regardless of the ComboBox or  Picker's size so I'll update it to follow that instead

Reply:
Hm...actually looking back on it, we never passed down the size prop to the Divider so it always defaulted to 'M'. In that case, I'll just update it so that it's always M regardless of the ComboBox or Picker's size.

Reply:
I think it should be fine? Since we are passing the size as 'M', it should always use the 'M' styles regardless of ComboBox or Picker